scripts/templates/train-template.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#SBATCH --account vjgo8416-mod-sim-2  # Only required if you are a member of more than one Baskerville project
```

Comment:
can you remove some of the boilerplate comments here?

Reply:
Done